assets/wizards/performance/views/push-notifications/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Performance Wizard Push Notifications screen.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { Component, Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { ToggleControl } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { TextControl, withWizardScreen } from '../../../../components/src';
+
+/**
+ * Description and controls for the Push Notification feature.
+ */
+class PushNotifications extends Component {
+	/**
+	 * Render.
+	 */
+	render() {
+		const { settings, updateSetting } = this.props;
+		return (
+			<Fragment>
+				<p>
+					{ __(
+						'You just published new content and you want to let everyone know? Why not send a push notification? We has an integrated connection to Firebase that lets you manage registered devices and send push notifications to all or selected devices!'
+					) }
+				</p>
+				<ToggleControl
+					label={ __( 'Enable Push Notifications' ) }
+					onChange={ checked => updateSetting( 'push_notifications', checked ) }
+					checked={ settings.push_notifications }
+				/>
+				{ settings.push_notifications && (
+					<Fragment>
+						<div className="newspack-performance-wizard_indented-block">
+							<p>{ __( 'This plugin uses Firebase Cloud Messaging as a messaging service.' ) }</p>
+							<ul>
+								<li>{ __( 'Go to Firebase Console' ) }</li>
```

Comment:
Can we make this a link to `https://console.firebase.google.com`?

Reply:
Done in https://github.com/Automattic/newspack-plugin/pull/141/commits/812992613a4ee56f242d3dcc37e61a7f2768ef24